### DISCUSSION TITLE
Option "allowPlugins" in GitHub Renovate App

### DISCUSSION THREAD

**Discussion by malteschlueter** [renovatebot/renovate#13679] [Request Help] [answered]
Is it possible to use the option "[allowPlugins](https://docs.renovatebot.com/self-hosted-configuration/#allowplugins)" with the [GitHub Renovate App](https://github.com/apps/renovate)?

**Comment by rarkins** [comment, marked as answer]
No, that's a self-hosted configuration option

**Comment by malteschlueter** [comment]
Damn it. :D 

Anyway, thank you very much for your fast answer and especially your work :)
